cmd/osm-crd-conversion-webhook/osm-crd-conversion-webhook.go

```diff
@@ -0,0 +1,197 @@
+// Package main implements the main entrypoint for osm-crd-conversion-webhook and utility routines to
```

Comment:
Can we rename the file and director to the app name `osm-crd-converter`, it would be implicit that this is a webhook.